cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -115,6 +115,13 @@
                             block__flush-top">
                     {{ render_stream_child(block) }}
                 </div>
+            {% elif 'post_preview_snapshot' in block.block_type %}
```

Comment:
2nd scenario after the conditionals got fixed.
I have 1 sublanding page --> 1 Browse Filterable Page --> 1 Document detail

get an undefined error `list object has no element 1` on the post_preview render function `{{ post_preview.render(posts[i], '', 0, pageurl(posts[i].parent())) }}`